components/search/SearchFacets.stories.js

```diff
@@ -0,0 +1,20 @@
+import { storiesOf } from '@storybook/vue';
+
+import SearchFacets from './SearchFacets.vue';
+
+storiesOf('Search page', module)
+  .add('Search facets', () => ({
+    components: { SearchFacets },
+    data() {
+      return { optionsType: { 'TYPE': { 'TEXT': 123456, 'VIDEO': 789, '3D': 10 } } };
```

Comment:
optionsType containing 'TYPE' seems redundant. If the facet keys are present anyway let's just use the same options everywhere.